### DISCUSSION TITLE
Issue about Submission

### DISCUSSION THREAD

**Discussion by KevinRoitero** [PaddlePaddle/PaddleSpatial#237] [BaiduKDDCup2022] [answered]
Team: Uniud
Submission time: 2022-07-04 19:25

I tried to submit a test run. My .zip folder contains only the "predict.py" and "prepare.py" files, formatted according to the instructions. 
When I try to submit I get the following error. Can you please have a look at the full log?
I did try to include the data folder and files but I keep getting the same error.

Thanks.

```
status: Error Occurred
message: Err: The preparation script, i.e. 'prepare.py', does NOT exist! ! (prova.zip)
```
![Screenshot at Jul 04 13-44-13](https://user-images.githubusercontent.com/27559491/177148392-26208054-d1e6-4a23-ae35-5d9193645909.png)

![Screenshot at Jul 04 13-47-43](https://user-images.githubusercontent.com/27559491/177148731-e39b58e0-9641-4314-858c-47988e5395c6.png)

**Comment by KevinRoitero** [comment, marked as answer]
I managed to make it work, the files needs to be compressed by selecting them individually and making sure there is no hidden file (like Mac OS ones) among them.

**Comment by zhiyansong** [comment]
I have also encountered the same problem, the error only shows the exception, may I ask whether your problem has been solved?

**Comment by KevinRoitero** [comment]
yes, you need to compress only .py files without any additional folder. The python files should be the in the root of the compressed file.

**Reply by GuibinChen92057** [reply]
I have compressed all .py files with a model.pth into a zip folder on my MacBook, but when I submit, I have still encountered with the same error. Could you possibly give more details?  If possible I would like an instruction in Chinese, many thx!

**Reply by ramber1836** [reply]
Please follow the Submission Manual (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md) and the example zipped file (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/tests/test-1.zip).
